<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="4f44e995-c6c8-40e9-ae01-5042c0f2cb64" name="Administrator@WIN-RNMUH95F9U1 2011-09-11 11:19:32" runUser="WIN-RNMUH95F9U1\Administrator" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="b356ad35-6e09-43b5-a51a-d3482c61513e">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Administrator_WIN-RNMUH95F9U1 2011-09-11 11_19_32">
      <DeploymentItem filename="C:\Projects\Urbana\Commerce\System.Web.Commerce\packages\EntityFramework.4.1.10715.0\lib\EntityFramework.dll" />
      <DeploymentItem filename="C:\Projects\Urbana\Commerce\System.Web.Commerce\System.Web.Commerce.Data\bin\Debug\System.Web.Commerce.Data.dll" />
      <DeploymentItem filename="C:\Projects\Urbana\Commerce\System.Web.Commerce\Dependencies\Common.Types.Schema.dll" />
      <DeploymentItem filename="C:\Projects\Urbana\Commerce\System.Web.Commerce\System.Web.Commerce\bin\Debug\System.Web.Commerce.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2011-09-11T11:19:32.0670156-04:00" queuing="2011-09-11T11:19:33.1490469-04:00" start="2011-09-11T11:19:33.2496328-04:00" finish="2011-09-11T11:20:03.6539297-04:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="Product_InsertTest" storage="c:\projects\urbana\commerce\system.web.commerce\system.web.commerce.data.tests\bin\debug\system.web.commerce.data.tests.dll" id="106d96df-1076-bcf3-6241-6ba6729b6e5e">
      <Execution id="e5df7511-b3f8-4d19-ae86-b865ba7573d0" />
      <TestMethod codeBase="C:/Projects/Urbana/Commerce/System.Web.Commerce/System.Web.Commerce.Data.Tests/bin/Debug/System.Web.Commerce.Data.Tests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="System.Web.Commerce.Data.Tests.ContextTests, System.Web.Commerce.Data.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Product_InsertTest" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="106d96df-1076-bcf3-6241-6ba6729b6e5e" executionId="e5df7511-b3f8-4d19-ae86-b865ba7573d0" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="e5df7511-b3f8-4d19-ae86-b865ba7573d0" testId="106d96df-1076-bcf3-6241-6ba6729b6e5e" testName="Product_InsertTest" computerName="WIN-RNMUH95F9U1" duration="00:00:29.5167774" startTime="2011-09-11T11:19:33.2808828-04:00" endTime="2011-09-11T11:20:03.6021719-04:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="e5df7511-b3f8-4d19-ae86-b865ba7573d0">
      <Output>
        <ErrorInfo>
          <Message>Test method System.Web.Commerce.Data.Tests.ContextTests.Product_InsertTest threw exception: 
System.Data.ProviderIncompatibleException: The provider did not return a ProviderManifestToken string. ---&gt; System.Data.SqlClient.SqlException: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message>
          <StackTrace>    at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning()
   at System.Data.SqlClient.TdsParser.Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandler, Boolean ignoreSniOpenTimeout, Int64 timerExpire, Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity)
   at System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, SqlConnection owningObject)
   at System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, Boolean redirectedUserInstance, SqlConnection owningObject, SqlConnectionString connectionOptions, TimeoutTimer timeout)
   at System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(SqlConnection owningObject, TimeoutTimer timeout, SqlConnectionString connectionOptions, String newPassword, Boolean redirectedUserInstance)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, Object providerInfo, String newPassword, SqlConnection owningObject, Boolean redirectedUserInstance)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnection owningConnection, DbConnectionPool pool, DbConnectionOptions options)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject)
   at System.Data.ProviderBase.DbConnectionPool.GetConnection(DbConnection owningObject)
   at System.Data.ProviderBase.DbConnectionFactory.GetConnection(DbConnection owningConnection)
   at System.Data.ProviderBase.DbConnectionClosed.OpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory)
   at System.Data.SqlClient.SqlConnection.Open()
   at System.Data.SqlClient.SqlProviderServices.UsingConnection(SqlConnection sqlConnection, Action`1 act)
   at System.Data.SqlClient.SqlProviderServices.UsingMasterConnection(SqlConnection sqlConnection, Action`1 act)
   at System.Data.SqlClient.SqlProviderServices.GetDbProviderManifestToken(DbConnection connection)
   at System.Data.Common.DbProviderServices.GetProviderManifestToken(DbConnection connection)
 --- End of inner exception stack trace ---
    at System.Data.Common.DbProviderServices.GetProviderManifestToken(DbConnection connection)
   at System.Data.Entity.ModelConfiguration.Utilities.DbConnectionExtensions.GetProviderInfo(DbConnection connection, DbProviderManifest&amp; providerManifest)
   at System.Data.Entity.DbModelBuilder.Build(DbConnection providerConnection)
   at System.Data.Entity.Internal.LazyInternalContext.CreateModel(LazyInternalContext internalContext)
   at System.Data.Entity.Internal.RetryLazy`2.GetValue(TInput input)
   at System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   at System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   at System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   at System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   at System.Data.Entity.Internal.Linq.InternalSet`1.ActOnSet(Action action, EntityState newState, Object entity, String methodName)
   at System.Data.Entity.Internal.Linq.InternalSet`1.Add(Object entity)
   at System.Data.Entity.DbSet`1.Add(TEntity entity)
   at System.Web.Commerce.Data.Tests.ContextTests.Product_InsertTest() in C:\Projects\Urbana\Commerce\System.Web.Commerce\System.Web.Commerce.Data.Tests\ContextTests.cs:line 64
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>